lib/Jsrt/CMakeLists.txt

```diff
@@ -52,3 +52,25 @@ target_include_directories (
     ../Runtime/Debug
     ../Parser
     )
+
+
+add_custom_target(ChakraCoreHeaders ALL
+  COMMAND ${CMAKE_COMMAND} -E make_directory "${CMAKE_BINARY_DIR}/include"
+  COMMAND ${CMAKE_COMMAND} -E copy_if_different
+    "${CMAKE_CURRENT_SOURCE_DIR}/ChakraCore.h"
+    "${CMAKE_BINARY_DIR}/include"
+  COMMAND ${CMAKE_COMMAND} -E copy_if_different
+    "${CMAKE_CURRENT_SOURCE_DIR}/ChakraCommon.h"
+    "${CMAKE_BINARY_DIR}/include"
+  COMMAND ${CMAKE_COMMAND} -E copy_if_different
+    "${CMAKE_CURRENT_SOURCE_DIR}/ChakraDebug.h"
+    "${CMAKE_BINARY_DIR}/include"
+  )
+
+if(MSVC OR WIN32)
```

Comment:
@kphillisjr I'm not saying we can't do that. The problem is that preprocessor flags we set and source files we do include on CMake xplat is not necessarily matching what we supposed to do for Windows. For the best ChakraCore experience on Windows, we better use msbuild with sln file.